adr/cdk-cli-lib.md

```diff
@@ -0,0 +1,414 @@
+# CDK CLI Library
```

Comment:
I love it! I think it would be great to add a `## Status` here, this will allow us to mark decisions as obsolete or superseded

Reply:
This is good, could be a separate PR though :)